File: src/main/java/com/pepebot/repository/TradeLogRepository.java
package com.pepebot.repository;

import java.util.List;

import org.springframework.data.jpa.repository.JpaRepository;

import com.pepebot.model.TradeLog;

public interface TradeLogRepository extends JpaRepository<TradeLog, String> {

	List<TradeLog> findTop20ByOrderByExecutedAtDesc();
}

File: src/main/kotlin/com/pepebot/repository/TradeLogRepository.kt
package com.pepebot.repository

import com.pepebot.model.TradeLog
import org.springframework.data.jpa.repository.JpaRepository

interface TradeLogRepository : JpaRepository<TradeLog, String> {
    fun findTop20ByOrderByExecutedAtDesc(): List<TradeLog>
}
